pages/integrations/output_framework.rst

```diff
@@ -14,10 +15,18 @@ outbound messages.
           version 3.3.3, thus an Enterprise license is required. See the 
           :doc:`Integrations Setup <setup>` page for more info.
 
-Enterprise Framework Outputs
-----------------------------
+.. toctree::
+   :titlesonly:
+   :hidden:
+
+   output_framework/output_tcp_raw
+   output_framework/output_tcp_syslog
+   output_framework/output_google_bigquery
```

Comment:
I personal would move the toctree at the end of the page for two reasons:

The order in navigation menu on the left:

![image](https://user-images.githubusercontent.com/404238/91958553-cbebbe80-ed07-11ea-9553-8d028a86573f.png)


Plus if we decide to make it clean visible this page does not need to get a re-arrange.

Reply:
I'm struggling a bit with this part.  If I move the toctree to the end of the page, it disappears from the sidebar (I suspect because it gets interpreted as being part of the last section, which is a subsection and would not appear in the TOC?).  Not giving up yet.